index.js

```diff
@@ -604,7 +609,22 @@ const runTestsUsingCommandLine = async () => {
  * @see https://on.cypress.io/module-api
  */
 const runTests = async () => {
+  const commandPrefix = core.getInput('command-prefix')
```

Comment:
This is more of a personal preference, and I think it makes code easier to read, but declaring all variables at the top of a function, instead of having them declared through the body of a function. I am more than happy to revert this refactor to the way things were, if others think that is best. 

I did this so I could read and understand what this function is doing.